locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po

```diff
@@ -130,7 +130,7 @@ msgstr "Bei Auswahl einfügen"
 #: prefs.js:194
 #, fuzzy
 msgid "Process primary selection"
-msgstr "Bei Auswahl einfügen"
+msgstr "Primäre Auswahl bearbeiten"
```

Comment:
What is the difference between the "primary selection" and a regular selection?

Reply:
https://wiki.archlinux.org/title/clipboard#Selections

Primary just means whatever is in your middle mouse click buffer.